_notebooks/2020-03-27-covid19-progress-projections.ipynb

```diff
@@ -0,0 +1,2126 @@
+{
```

Comment:
<ul><li>"reported total deaths to total cases percentage" needs to be "reported total deaths divided by total cases."</li><li>The field "<strong>Estimated new cases in 5 days</strong>." doesn't have a definition.  Also should be "in last 5 days" not "in 5 days"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/github/covid19-dashboard/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>